docs/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,56 @@
+|"Unable to register watch service, configuration refresh will not work" | Path Directory does not exists, File Permission disagred with request, etc.                        |
+|"Unable to get maxRepaired for {<table_reference>}"                     | Cassandra Node Overloaded, Instance Not Found, Table does not Exists Anymore, etc.                |
+
+*Futhermore, one common problem is when repair is taking too much time/is overdue. This can be caused by too many vnodes, nodes going down while holding a lock and Cassandra nodes being down.*
+
+## Metrics
+If you have Cassandra metrics reporting to a centralized location such as Graphite or Grafana you can typically use those to narrow down the problem. But ecChronos can be monitored, as described in the [repair monitoring documentation](REPAIR_MONITORING.md). Using metrics, it can be possible to identify some possible problems in the cluster and evaluete some basic things, like overload, network issues, cassandra rejections, etc.
+
+## Use ECCTOOL
+ecctool is a command line utility which can be used to perform actions towards a local ecChronos instance. The actions are implemented in form of subcommands with arguments. All visualization is displayed in form of human-readable tables.
+
+To have more information about ecctool you can check the [ecctool examples](ECCTOOL_EXAMPLES.md) and [ecctool documentation](autogenerated/ECCTOOL.md)
+
+## Cassandra Troubleshooting
+As ecChronos is a client that sends requests to Cassandra to carry out repairs, sometimes it may happen that the Cassandra node does not respond for some reason, so Cassandra itselt have a great set of tools an strategys to troubleshoot an issue, which can be found in the [Cassandra Troubleshooting Documentation](https://cassandra.apache.org/doc/latest/cassandra/troubleshooting/index.html).
+
+## Report a Bug
+If using the tools and performing debugging were not enough to determine the root cause of the problem, consider opening an ISSUE so that the ecChronos team can investigate the problem and provide support, for that, you can check the [Contributing Guide](CONTRIBUTING.md).
+
+Also, before opening an ISSUE, gather log files, metrics and tests carried out so that the investigation can be clearer and open the ticket with the following format:
+
+```
+Title: A concise and descriptive summary of the Problem
+
+Description:
+
+Give us a descriptions that can asnwer those questions:
+
+1- Does the problem persist (Y/N):
+2- Reproducible (Y/N):
+3- How to reproduce the problem?
+
+Detailed Description:
+• What is happening?
+• What did you expect to happen?
+• What have you tried? 
+• Was the problem detected during an upgrade or downgrade procedure? 
+• Have you checked the ecChronos documents?
+• What do YOU think is the issue (optional)?
+• Anything else that is worth being noted (optional)?
+```
+
+Remember to attach log files and command results if possible.
```

Comment:
"Always attach log files and command results."

Skip the "if possible" part. They should always include all they have (and feel the "pressure" to do so), otherwise we might end up with a lot of issue with vague descriptions.